File: app/src/main/java/br/com/onibusapp/onibusapp/domain/Linha.java
package br.com.onibusapp.onibusapp.domain;

import java.io.Serializable;

/**
 * Created by diego on 15/07/2018.
 */

public class Linha implements Serializable {

    private String numero;
    private Integer codigoEmpresa;


    public Linha(String numero, Integer codigoEmpresa) {
        this.numero = numero;
        this.codigoEmpresa = codigoEmpresa;
    }

    public String getNumero() {
        return numero;
    }

    public void setNumero(String numero) {
        this.numero = numero;
    }

    public Integer getCodigoEmpresa() {
        return codigoEmpresa;
    }

    public void setCodigoEmpresa(Integer codigoEmpresa) {
        this.codigoEmpresa = codigoEmpresa;
    }

}

File: app/src/main/java/br/com/onibusapp/onibusapp/data/dominio/Linha.kt
package br.com.onibusapp.onibusapp.data.dominio

import java.io.Serializable

/**
 * Created by diego on 15/07/2018.
 */

data class Linha (var id: Int = 0,
                  var linha: String = "",
                  var nome: String = "",
                  var codigoEmpresa: Int = 0,
                  var dias: String = "") : Serializable
